journal/updated/01-journal.adoc

```diff
@@ -1,146 +1,152 @@
+was preaching, while I wrote down his words after him. In his sermon he said,
 "`The time would come that there would be no need of the scriptures,
-any more than another book;`" at which I very much stumbled;
-and after the meeting I asked him, when would that time be?
-He said, when the Lord would make a new covenant with his people,
+any more than another book,`" at which I very much stumbled,
+and after the meeting I asked him when that time would be.
+He said, "`when the Lord would make a new covenant with His people,`"
 as it is said in Jeremiah 31:33-34,
 "`I will make a new covenant with the house of Israel; after those days, says the Lord,
-I will put my law in their inward parts, and write it in their hearts,
-and will be their God, and they shall be my people.
-And they shall teach no more every man his neighbour, and every man his brother, saying,
-"`Know the Lord; for they shall all know me,
+I will put My law in their inward parts, and write it in their hearts,
+and I will be their God, and they shall be My people.
+And they shall teach no more every man his neighbour,
+and every man his brother, saying, '`Know the Lord;`' for they shall all know Me,
 from the least of them unto the greatest of them, says the Lord;
 for I will forgive their iniquity,
-and I will remember their sin no more.`" It seems that he knew not that day come then,
-though he was a great preacher. I thought it would then be a happy day,
-when God would be the teacher of his people, himself;
-that we need not teach every man his neighbour, or his brother, saying,
-"`Know the Lord;`" but that we should all know him, from the least to the greatest.
-This day we knew not then, for all our preaching and long prayers;
-though the Lord did then beget true hungerings and thirstings in our souls after him.
-We had great love and zeal, and desired the knowledge of the truth, as it is in Jesus.
-Sometimes I have said, this was but like Jacob`'s dream, when he awoke and said,
+and I will remember their sin no more.`"
+It seems this man knew not then the day he spoke of,
+though he was a great preacher. I thought it would be a happy day,
+when God Himself would be the teacher of His people,
+so that we need not teach every man his neighbour, or his brother, saying,
+"`Know the Lord;`" but that we would all know Him, from the least to the greatest.
+
+This day we knew not then, despite all our preaching and long prayers,
+though the Lord did then beget true hungerings and thirstings in our souls after Him.
+We had great love and zeal, and desired the knowledge of the truth as it is in Jesus.
+Sometimes I have said this was but like Jacob`'s dream, when he awoke and said,
 "`Surely the Lord is in this place, and I knew it not.`"
-And indeed we knew not the Lord, as we ought to have done; namely, by his light, grace,
+And indeed we knew not the Lord as we ought to have done, namely, by His light, grace,
 and spirit shining in our hearts,
```

Comment:
I usually capitalize the word Spirit whenever it has to do with the Spirit of the Lord (although the Quakers normally didn't)